import_export/resources.py

```diff
@@ -683,7 +683,7 @@ def after_import_instance(self, instance, new, row_number=None, **kwargs):
         """
         pass
 
-    def import_row(self, row, instance_loader, using_transactions=True, dry_run=False, raise_errors=False, **kwargs):
+    def import_row(self, row, instance_loader, using_transactions=True, dry_run=False, **kwargs):
```

Comment:
It would be best to deprecate `raise_errors` rather than remove, because some users may have overridden this method and may still be using it.  Although I can't think of a clean way of doing this.